desktop-widgets/importgps.cpp

```diff
@@ -0,0 +1,329 @@
+// SPDX-License-Identifier: GPL-2.0
+#include "ui_importgps.h"
+#include "desktop-widgets/importgps.h"
+#include "desktop-widgets/locationinformation.h"
+#include "core/dive.h"
+
+#include <time.h>
+#include <QFileDialog>
+#include <QProcess>
+#include <QDateTime>
+#include <QDate>
+#include <stdlib.h>
+#include <stdio.h>
+#include <QDebug>
```

Comment:
I'm always a bit weary about QDebug includes. Is it needed?

Reply:
done.